libass/ass_fontselect.c

```diff
@@ -31,7 +31,11 @@
 #include <limits.h>
 #include <ft2build.h>
 #include <sys/types.h>
+#if defined(_WIN32) && !defined(__CYGWIN__)
+#include <Windows.h>
```

Comment:
The previously used lowercase variant is better as it matches the preexisting `#include <windows.h>` and `#include <winapifamily.h>` from `config.ac` and the DirectWrite font provider. (Several instances of this)

Reply:
And it’s not just for superficial consistency: when cross-compiling on Linux, the names are usually case-sensitive and only the lowercase version exists.